### DISCUSSION TITLE
Circuit from invert signal on the TM1914 chipset

### DISCUSSION THREAD

**Discussion by dschaulet** [Makuna/NeoPixelBus#515] [Q&A] [answered]
Hi!

I´m trying to find a way to turn on a led strip using chip TM1914. (https://pt.aliexpress.com/item/32734325240.html)

I see a issue with Makuna (#474 ) where he ask for @oraclesod use a circuit between the pin of board and the led strip to invert the signal, Someone have a example of a circuit and the components that I need?

Thanks a lot!

**Comment by Makuna** [comment, marked as answer]
Simple interweb search ...

https://forum.arduino.cc/t/transistor-inverter-logic/178556

**Reply by dschaulet** [reply]
On more time thanks for you fast response an attention Michael!
